DSharpPlus.Commands/Processors/SlashCommands/Attributes/InteractionAutoCompleteProviderAttribute.cs

```diff
@@ -7,7 +7,7 @@ namespace DSharpPlus.Commands.Processors.SlashCommands.Attributes;
 using Microsoft.Extensions.DependencyInjection;
 
 [AttributeUsage(AttributeTargets.Parameter, Inherited = false, AllowMultiple = false)]
-public class SlashAutoCompleteProviderAttribute(Type autoCompleteType) : Attribute
+public class InteractionAutoCompleteProviderAttribute(Type autoCompleteType) : Attribute
```

Comment:
Autocomplete is currently only available for slash commands. Should remain as `SlashAutoCompleteProviderAttribute` imo.